Neos.Flow/Configuration/Settings.Http.yaml

```diff
@@ -24,9 +24,15 @@ Neos:
       # request (default).
       baseUri: NULL
 
+      # Defines the PSR-15 middlewares that a request goes through in a flow
+      middlewares:
+        'mvc-dispatch':
```

Comment:
Maybe call this only `dispatch` as mvc is an implementation detail of the controller.

Reply:
But isn't the fact the request gets dispatched to a **Controller** already part of the MV**C** approach? I would have considered a different approach to be achieved by a different dispatcher - but maybe you are right. OTOH at least it lives in the MVC namespace, so I *guess* the name is okay-ish?

For reference: https://blog.ircmaxell.com/2014/11/alternatives-to-mvc.html
I mean, I never heard of most of those other patterns and following the conclusion, they probably are all the same "MVC" aproach under the coat (a triad of logic, represent, interact), just with little different focus. Iff we ever introduce a really different pattern to handle a request, we might name it `sna-dispatch` (**S**uper **N**ew **A**cronym) - or maybe not even "dispatch" at all?

One possible use-case could be an [Neos.EventSourcing](https://github.com/neos/Neos.EventSourcing) specific "dispatcher" or "commandhandler"?

Reply:
MVC is distinct from dispatching that only is intermingled in the current implementation. We could consider to clean this up in future and extract the dispatching from there. 

I also thought this would be a safe assumption now think that the core is indeed a pure action that is given a request and that returns a response. Wether there is a model or a view or any other pattern involved is nothing the dispatcher has to care about. 

Offcourse this can be implemented in special controller base classes like actionController like it is in flow as all argumentMapping and validation is already done in actionController and not AbstractController. 

Obviously this all does not affect this pr. The only question we have to discuss here is which name would be best and fitting for future developments. I personally think that `dispatch` is best fitting from the perspective of the http-middleware-chain.

Reply:
Maybe the name `controller-dispatch` would be fitting as it only assumes a derived class of AbstractController is called.

Reply:
> The only question we have to discuss here is which name would be best and fitting for future developments.

Right and I guess you are right. `dispatch` covers the exact case of this very middleware - no matter if it will dispatch to some MVC, command, or other pattern. In fact, if we (or a third party) chooses to switch this, it makes most sense to **replace** this middleware anyway, instead of prepending to it and then short-circuiting it. So the super generic name `dispatch` is fitting for all means and purposes of this "ever last middleware". At least more than `mvc-dispatch`.

Reply:
```suggestion
        'dispatch':
```